build-cs/.gitignore

```diff
@@ -0,0 +1 @@
+vendor/
```

Comment:
Should this not be in the root gitgnore?

Reply:
Both works, just a matter of preference I guess.

Reply:
I would not think to look for this file. Let just have one file.